### DISCUSSION TITLE
invalid input syntax for type bigint: "str" for Array

### DISCUSSION THREAD

**Discussion by danilsizov** [supabase/supabase#11302] [Questions] [answered]
I have this code to get branches by tag. Tags is an array of strings.

```
let id = 'str'
let {data: branches, error, count} = await supabase
            .from('branches')
            .select('*')
            .contains('tags', [id])
            .range(from, to);
```
But I got this error message:
```
{
  code: '22P02',
  details: null,
  hint: null,
  message: 'invalid input syntax for type bigint: "Kukai"'
}
```

**Comment by GaryAustin1** [comment, marked as answer]
What are from and to?  Could one of those be "Kukai"?
Also when you use range, you need to use .order() on a column.
There is no default order in Postgres and it can change between calls.

**Reply by danilsizov** [reply]
from and to are numbers

Here is full code of function:
`const getBranchesByTag = async function(id, from=0, to=999){
    try{
        let {data: branches, error, count} = await supabase
            .from('branches')
            .select('*')
            .contains('tags', [id])
            .range(from, to);
        console.log(error)
        return { branches: branches, count: count }
    } catch (err){
        console.log(err)
    }
}`

**Reply by danilsizov** [reply]
Ok, I added .order(), and it's working now, thank you so much!
